docs/install.md

```diff
@@ -23,7 +23,7 @@ _Note: You need to install all the dependencies in order to use the extension._
 - [Python VS Code extension](https://marketplace.visualstudio.com/items?itemName=ms-python.python)
```

Comment:
(That's not on the scope of this PR, so don't worry!) I just realized we don't have instructions for building and installing the extension locally, was there any reason for that?

Reply:
We do, but I think it was hard to find/understand because of the wording so I updated it in the next commit. Let me know what you think. I wasn't sure if I should add a whole "Contributing" section or if just adding the doc to set up in the "Useful Links" section is good enough.